include/inexor/vulkan-renderer/wrapper/image.hpp

```diff
@@ -12,11 +12,11 @@ class Device;
 /// @brief RAII wrapper class for VkImage.
 class Image {
     const wrapper::Device &m_device;
-    VmaAllocation m_allocation;
+    VmaAllocation m_allocation{VK_NULL_HANDLE};
     VmaAllocationInfo m_allocation_info;
-    VkImage m_image;
-    VkFormat m_format;
-    VkImageView m_image_view;
+    VkImage m_image{VK_NULL_HANDLE};
+    VkFormat m_format{VK_FORMAT_UNDEFINED};
```

Comment:
Format one not needed since it's initialised by constructor.

Reply:
The problem is that if we add or change the constructor and they do not hurt or?

Reply:
@yeetari ping

Reply:
This is the same issue as above, so not sure what you want to do - honestly it doesn't really matter so I'll approve anyway.